Snippets/ASP/ASTP_3/Behaviors.cs

```diff
@@ -84,34 +84,39 @@ public override async Task Invoke(IIncomingLogicalMessageContext context, Func<T
 
 class OrderIdAsPartitionKeyBehavior : Behavior<IIncomingLogicalMessageContext>
 {
-    public OrderIdAsPartitionKeyBehavior(IProvidePartitionKeyFromSagaId partitionKeyFromSagaId)
-    {
+    private static readonly ILog Log = LogManager.GetLogger<OrderIdAsPartitionKeyBehavior>();
```

Comment:
I thought we did not use explicit access modifiers? (I'm all for adding them, as I've trying to retrain my brain *not* to add them everywhere and still fail at that).